src/test/scala/org/mbari/vars/annotation/api/v1/ImagedMomentV1ApiSpec.scala

```diff
@@ -153,4 +162,104 @@ class ImagedMomentV1ApiSpec extends WebApiStack {
     }
   }
 
+  it should "count and find with images" in {
```

Comment:
Modify the test to add two images at the same imagedMoment to verify the count returned is 1.

Reply:
Test now creates 10 images at the same time, then asserts only one imaged moment exists with the same number of images.